mobile/src/people/home-people-screen.tsx

```diff
@@ -18,10 +19,13 @@ const navBar =  new TopNavBar('People');
 
 class HomePeopleScreenImpl extends React.Component<PeopleScreenProps> {
     public static navigationOptions = navBar.configurate();
-
+    
     public render() {
+        
         return (
-            <Text>PeopleScreen. There are { this.props.departments.length} departments</Text>
+            <View style={{ flex: 1 }}>
+                <TopTabBarNavigator />
```

Comment:
TopTabBarNavigator should not be called as such. It works only in scope of peoples scene. Rename it to PeopleScreenNavigator or something like that.